app/controllers/forest_liana/actions_controller.rb

```diff
@@ -5,7 +5,7 @@ def get_smart_action_hook_request
       begin
         params[:data][:attributes]
       rescue => error
-        FOREST_LOGGER.error "Smart Action hook request error: #{error}"
+        FOREST_LOGGER.error "Smart Action hook request error: #{error}", error
```

Comment:
From my point of view `FOREST_LOGGER` purpose is to print logs and only do so.

I would not feel it strange to read:
```suggestion
        FOREST_LOGGER.error "Smart Action hook request error: #{error}"
        FOREST_REPORTER.report error
```